OktaAuthNative.xcodeproj/project.pbxproj

```diff
@@ -191,7 +189,6 @@
 				F5CC2ADA21D5056D00649C33 /* OktaAPITests.swift */,
 				2F8D79F521E5FAD200CFDF12 /* OktaModelsTests.swift */,
 				A1FCB127220908B70026F522 /* AuthenticationClientTests.swift */,
-				2FF5C86621D3BA1900EECA75 /* OktaAuthiOSIntegrationTests.swift */,
```

Comment:
Don't these files still exist? Why are they being removed?

Reply:
I temporary removed end-to-end testing as it requires environment config on travis side. We will definitely return this test case once we add required configuration